downloadCurrentVersion.js

```diff
@@ -13,7 +13,7 @@ function getVersionList (cb) {
   console.log('Retrieving available version list...');
 
   var mem = new MemoryStream(null, { readable: false });
-  https.get('https://raw.githubusercontent.com/ethereum/solc-bin/gh-pages/bin/list.json', function (response) {
+  https.get('https://solc-bin.ethereum.org/bin/list.json', function (response) {
```

Comment:
`https.get()` won't handle HTTP redirects as far as I know. Unfortunately it looks like you need an external library for that. Any suggestions? Is it ok to add a new dependency just for that?

Reply:
Oh really? Yeah, please either add the most lightweight library or write a simple function that follows at most 3 redirect or something.

Reply:
I'll do it in a separate PR not to block this one. Right now we don't have any redirects in `solc-bin` anyway.